system/lib/compiler-rt/exception_builtins.S

```diff
@@ -0,0 +1,6 @@
+.globl __trap
+
+__trap:
+  .functype __trap() -> ()
```

Comment:
I think you can just write this in C as `void __trap() { __builtin_trap() }`

Reply:
Done